UnityProject/Assets/Scripts/UI/Core/OptionsMenu/SoundOptions.cs

```diff
@@ -142,7 +159,7 @@ public void ResetDefaults()
 					AudioManager.MusicVolume(0.2f);
 					AudioManager.TtsVolume(0.2f);
 					AudioListener.volume = 1;
-					AudioManager.CommonRadioChatter(false);
+					AudioManager.CommonRadioChatter(true);
```

Comment:
wasn't agreed upon for it to be false?

Reply:
I didn't agree to anything like that?
Why would I agree to disabling a feature that I added

Reply:
I think it should be off by default